instrumentation/opentelemetry-instrumentation-urllib/src/opentelemetry/instrumentation/urllib/__init__.py

```diff
@@ -183,15 +183,17 @@ def _instrumented_open_call(
             SpanAttributes.HTTP_URL: url,
         }
 
+        span_attributes = {
```

Comment:
There is already a dict with the exact same fields called `labels`. Maybe we can just use that.